src/test.js

```diff
@@ -0,0 +1,14 @@
+const action = {
```

Comment:
Consider renaming this file to "example.js"

Reply:
done